### PR TITLE
Fix response body dump for 2.x.x version

### PR DESCRIPTION
**Pull request by evstratbg**
fixed uvicorn `Response content longer than Content-Length` error, when an endpoint returns just a status code

```
@router.delete("/")
async def delete():
    return Response(status_code=status.HTTP_204_NO_CONTENT)
```

please note, this is a branch from fae0295 to hotfix 2.x.x version, so I need a separate branch for that. Could you create it? Otherwise there are conflicts

[Here](https://github.com/zmievsa/cadwyn/pull/106) is an MR for the main

### CODE REVIEW

File: cadwyn/structure/versions.py [hunk 1]
```diff
@@ -446,7 +446,10 @@ async def _convert_endpoint_response_to_version(
             path,
             method,
         )
-        if isinstance(response_or_response_body, FastapiResponse):
+        if isinstance(response_or_response_body, FastapiResponse) and response_info.body:
```
Comment (zmievsa) [inline cadwyn/structure/versions.py:449]: Please, let's move the second part of this `if` down in the same manner as in `main`

File: cadwyn/structure/versions.py [hunk 2]
```diff
@@ -446,7 +446,10 @@ async def _convert_endpoint_response_to_version(
...
+            # None is a valid response body, when a user returns just a status code
+            # dumping it will result in a 'null' string, which doesnt correlate with the content-length
```
Comment (zmievsa) [inline cadwyn/structure/versions.py:451]: Let's make it consistent with `main`'s comments.

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,7 +3,11 @@
 All notable changes to this project will be documented in this file.
 Please follow [the Keep a Changelog standard](https://keepachangelog.com/en/1.0.0/).
 
-## [Unreleased]
+## [2.3.5]
+
+### Fixed
+
+* do not json.dumps the body while converting endpoint response to version if body is empty
```
Comment (zmievsa) [inline CHANGELOG.md:10]: Let's make it consistent with `main`

**Review by zmievsa — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (evstratbg) [reply CHANGELOG.md]: fixed

File: cadwyn/structure/versions.py (hunk 2 above)
Comment (evstratbg) [reply cadwyn/structure/versions.py]: fixed

File: cadwyn/structure/versions.py (hunk 1 above)
Comment (evstratbg) [reply cadwyn/structure/versions.py]: fixed

**Pull request merged**